applications/smart-distancing/face_mask_recognition/_init_path.py

```diff
@@ -0,0 +1,13 @@
+# Set up paths for the Face Mask Recognition
```

Comment:
Please create a Readme for `/face_mask_recognition` and explain the intend of this module, expected results and a brief instruction.

Reply:
Thanks @alpha-carinae29 for feedbacks. I will do some changes based on your comments.